en/README.md

```diff
@@ -4,20 +4,28 @@
 Retro Gadgets is a gadget creation station where you invent, build, solder, code and customise your own electrifying gadgets then share them with the world.
 
 This repository contains the documentation for doing anything code-related in Retro Gadgets.  
-<a href="https://docs.retrogadgets.game" target="_blank">You can find the official documentation here, although it might not be of much use (yet)</a>
+You can find [the official documentation here](https://docs.retrogadgets.game), although it might not be of much use (yet)
 
-* <a href="https://store.steampowered.com/app/1730260/Retro_Gadgets/" target="_blank">Wishlist the game here!</a>
-* <a href="https://discord.com/invite/np6FUDjt9E" target="_blank">Join the Official Discord Server!</a>
-* [Learn how you can contribute by opening a pull request!](./docs/github/Contributing.md)</a>
+* [Buy the game here!](https://store.steampowered.com/app/1730260/Retro_Gadgets/)
+* [Join the Official Discord Server!](https://discord.com/invite/np6FUDjt9E)
+* [Learn how you can contribute by opening a pull request!](./docs/github/Contributing.md)
+
+### Coding gadgets
+Retro Gadgets uses Luau to code gadgets, which is basically Lua but it [has some differences](https://luau-lang.org/compatibility). You can still use resources for learning Lua and apply that knowledge to Luau. Here are some places to learn Lua (and Luau):
```

Comment:
Slight grammatical change, feel free to resolve this chain if you'd like to ignore it.
```suggestion
Retro Gadgets uses Luau to code gadgets, which is basically Lua but [with some differences](https://luau-lang.org/compatibility). You can still use resources for learning Lua and apply that knowledge to Luau. Here are some places to learn Lua (and Luau):
```